### PR TITLE
[example hello] Use `println!`, not `log::info!`.

### LINKED ISSUE #3503: Hello example does nothing
**Description**
[The `hello` example](https://github.com/gfx-rs/wgpu/tree/master/wgpu/examples/hello) does not print any output.

**Repro steps**
`cargo run --example=hello`

**Expected vs observed behavior**
According to the example's readme, there should be command-line output including `Available adapters:`. Instead, there is only the output from Cargo (on a retry: `Finished dev [unoptimized + debuginfo] target(s) in 0.39s` and <code>Running \`target\debug\examples\hello.exe\`</code>), the binary itself prints nothing.

**Platform**
Windows 11 Pro version 22H2 (build 22621.1265), `wgpu` commit e18520078a3eab5a5fc6beee974033e8a98822f3.

### PR DESCRIPTION
**Pull request by jimblandy**
Fixes #3503.

### CODE REVIEW

File: examples/src/hello/mod.rs [hunk 1]
```diff
@@ -16,7 +16,7 @@ async fn run() {
             .unwrap()
     };
 
-    log::info!("Selected adapter: {:?}", adapter.get_info())
+    println!("Selected adapter: {:?}", adapter.get_info())
```
Comment (grovesNL) [inline examples/src/hello/mod.rs]: `println` won't output anything on the web/wasm targets, but log does

**Review by ErichDonGubler — DISMISSED** [review]
AFAIK, `println!(…)` doesn't exist in `wasm` targets. I think it's better to adjust the `env_logger` initialization to log at `INFO` or higher for `wgpu_examples::hello`:

```rs
env_logger::builder()
	.filter(Some(module_path!()), log::LevelFilter::Info)
	.parse_default_env()
	.init();
```

**Comment by ErichDonGubler** [comment]
@jimblandy: I've pushed a `fixup!` (https://github.com/gfx-rs/wgpu/pull/5905/commits/5d57f0b5e2d4d944b86860b5cdbb25f73bf257ae) that should implement my feedback. Feel free to `git rebase -i --autosquash trunk` and rework the commit message if you want to accept it; I'll approve and merge if you do. 🙂

File: examples/src/hello/mod.rs (hunk 1 above)
Comment (ErichDonGubler) [reply examples/src/hello/mod.rs]: Tentatively resolved with https://github.com/gfx-rs/wgpu/pull/5905#issuecomment-2206748367 (https://github.com/gfx-rs/wgpu/pull/5905/commits/5d57f0b5e2d4d944b86860b5cdbb25f73bf257ae).

Comment (jimblandy) [reply examples/src/hello/mod.rs]: Ah, I figured there was some reason like this - and that if I filed a PR someone would tell me what it was:D

**Review by nical — APPROVED** [review]

**Pull request merged**